eng/Versions.props

```diff
@@ -42,7 +42,7 @@
     <_MicrosoftWebWebView2Version>1.0.1264.42</_MicrosoftWebWebView2Version>
     <!-- GLIDE - the android maven artifact in /src/Core/AndroidNative/maui/build.gradle -->
     <!-- must be kept in sync with the binding library version to it here: -->
-    <_XamarinAndroidGlideVersion>4.13.2.1</_XamarinAndroidGlideVersion>
+    <_XamarinAndroidGlideVersion>4.13.2.2</_XamarinAndroidGlideVersion>
```

Comment:
When we change the 4th number, it means it is a C#-only change. In this case, I think we bumped the version of the Android workload and rebuilt a lot of packages.

But then these look like they are out of date even now:

https://github.com/dotnet/maui/blob/93aeab24f3a024a6715028d0b21a9240591f8246/src/Core/AndroidNative/maui/build.gradle#L29-L31

These should all be 4.13.0?
* https://mvnrepository.com/artifact/com.github.bumptech.glide/glide/4.13.0
* https://mvnrepository.com/artifact/com.github.bumptech.glide/annotations/4.13.0
* https://mvnrepository.com/artifact/com.github.bumptech.glide/compiler/4.13.0